contracts/rate_oracles/GlpRateOracle.sol

```diff
@@ -130,27 +130,26 @@ contract GlpRateOracle is BaseRateOracle, IGlpRateOracle {
     {
         // get last observation
         uint216 lastIndexRay = observations[oracleVars.rateIndex].observedValue;
-        require(lastIndexRay > 0, "No previous observation");
 
         // calculate rate increase since last update
         uint256 cumulativeRewardPerToken = rewardTracker
             .cumulativeRewardPerToken();
+
+        if (cumulativeRewardPerToken < lastCumulativeRewardPerToken) {
```

Comment:
this in theory should never happen right?

Reply:
in theory, but it's an external contract so we at least check